src/core/Wizard/Wizard.test.tsx

```diff
@@ -7,6 +7,14 @@ import React from 'react';
 import { Wizard } from './Wizard';
 
 describe('<Wizard />', () => {
+  afterEach(() => {
+    // cleanup tippy after every test, so it does not stay in the dom
+    const tippy = document.querySelector('[data-tippy-root]') as HTMLElement;
+    if (!!tippy) {
+      tippy.remove();
+    }
```

Comment:
nit:

```suggestion
    document.querySelector('[data-tippy-root]')?.remove();
```

although, I'm not sure why it even stays in the dom

Reply:
changed